openhtf/io/test_record.py

```diff
@@ -29,6 +29,7 @@
 from openhtf.util import logs
 
 _LOG = logging.getLogger(__name__)
+CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
```

Comment:
so what's the intended way for a user to 'Override' this?  immediately after import via `test_record.CAPTURE_SOURCE_CODE = False`?